ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java

```diff
@@ -3104,6 +3112,15 @@ Seems much cleaner if each stmt is identified as a particular HiveOperation (whi
     return lockComponents;
   }
 
+  public static boolean isCTASOperation(List<LockComponent> lockComponents, HiveConf conf) {
+    boolean isCtas = false;
+    for (LockComponent lock : lockComponents) {
+      if (lock.getOperationType().name().equals(OperationType.CTAS.name()) &&
+              conf.getBoolVar(ConfVars.HIVE_ACID_CHECK_FOR_CONCURRENT_CTAS_ENABLED))
```

Comment:
why not use isNoRenameCtasEnabled/isExclusiveCTASEnabled helper method

Reply:
Done.